frontend/app/features/addEvent/controllers/addEventController.js

```diff
@@ -3,32 +3,32 @@
 
 
     function addEventController($scope, $rootScope, datePicker, addEventAPIService) {
-       $scope.datePicker = datePicker;
+        $scope.datePicker = datePicker;
         $scope.place = {};
 
         $scope.search = function() {
-            $scope.apiError = false;
-            $rootScope.search($scope.searchPlace)
-              .then(function(res) { // success
-                       $rootScope.addMarker(res);
-                      $scope.place.name = res.name;
-                      $scope.place.lat = res.geometry.location.lat();
-                      $scope.place.lng = res.geometry.location.lng();
-                  },
-                  function(status) { // error
-                      $scope.apiError = true;
-                      $scope.apiStatus = status;
-                  }
-              );
-        }
-        // map.init();
+                $scope.apiError = false;
+                $rootScope.search($scope.searchPlace)
+                    .then(function(res) { // success
+                            $rootScope.addMarker(res);
+                            $scope.place.name = res.name;
+                            $scope.place.lat = res.geometry.location.lat();
+                            $scope.place.lng = res.geometry.location.lng();
+                        },
+                        function(status) { // error
+                            $scope.apiError = true;
+                            $scope.apiStatus = status;
+                        }
+                    );
+            }
+            // map.init();
 
         $scope.add = function() {
             var eventTmp = {};
             eventTmp.title = $scope.title;
             eventTmp.desc = $scope.description;
             eventTmp.date = $scope.datePicker.dt;
-            eventTmp.place = $scope.place;
+            eventTmp.place = $scope.place.name;
             eventTmp.report = null;
             addEventAPIService.add(eventTmp);
```

Comment:
You can combine properties into one object. 
`ng-model="event.description"
ng-model="event.description"
`
And then you add function in controller will be smaller and more readable
`  $scope.add = function() {
     addEventAPIService.add($scope.event);
}`